frontend/src/lib/components/Errors/ErrorDisplay.tsx

```diff
@@ -63,15 +66,32 @@ function StackTrace({ rawTrace }: { rawTrace: string }): JSX.Element | null {
 }
 
 function ChainedStackTraces({ exceptionList }: { exceptionList: ExceptionTrace[] }): JSX.Element {
+    const [showAllFrames, setShowAllFrames] = useState(false)
```

Comment:
Should this be hoisted up to the logic or somewhere? I could imagine it's pretty annoying to have to toggle it on every exception you view if that's your intention

Reply:
I wasn't sure, but in general I wouldn't _expect_ so, the full stack trace is very rarely useful, hence all our effort to figure out what's in app and what's not. I think the only time I toggled this in sentry was when trying to see what all they offer here 😅  (so never for debugging in 3yrs). But no strong feelings, happy to persist in a logic as well

Reply:
Let's go with it as is and let customers tell us if they want something else

Reply:
Another perfectly reasonable thing here is to just exclude it 🤷 If neither of us have ever needed it begs the question whether we want it as a feature

Reply:
reasonable, I'll ship it and add an action to track how often its used :P